Team6220_2021/src/main/java/org/firstinspires/ftc/team6220_2021/RedParkBlockPLaceLeague2.java

```diff
@@ -4,9 +4,8 @@
 import com.qualcomm.robotcore.hardware.DcMotor;
 import com.qualcomm.robotcore.hardware.DcMotorSimple;
 import com.qualcomm.robotcore.hardware.Servo;
-
 @Autonomous(name = "RedBlockPlace", group = "Autonomous")
-public class RedParkBlockPLaceLeague1 extends MasterOpMode{
+public class RedParkBlockPLaceLeague2 extends MasterOpMode{
```

Comment:
Why doesn't this file extend from MasterAutonomous class instead of MasterOpMode class?